### ISSUE TITLE
Default Images Not Shown

### ISSUE THREAD

**Issue by prin53** [BruelAndKjaer/Chafu#15]
![screenshot](https://cloud.githubusercontent.com/assets/4447833/19201048/b0936062-8cd4-11e6-8190-7b03b71e69c6.png)

Steps to reproduce:
1. Remove library reference from sample.
2. Add `Chafu` package to the sample.
3. Run.

**Comment by Cheesebaron** [comment]
Yeah, it is a problem with building on AppVeyor with is not attached to a mac. So the resulting DLL's does not have images bundled.
